datasets/austin_waste/_terraform/waste_and_diversion_pipeline.tf

```diff
@@ -15,22 +15,25 @@
  */
 
 
-resource "google_bigquery_table" "cpsaat18" {
+resource "google_bigquery_table" "waste_and_diversion" {
   project    = var.project_id
-  dataset_id = "bls"
-  table_id   = "cpsaat18"
```

Comment:
This PR is for the `austin_waste` dataset, but seems like the Terraform files for the `bls` dataset was modified?